### ISSUE TITLE
create indexes

### ISSUE THREAD

**Issue by Morriz** [omar-dulaimi/prisma-yup-generator#5]
## Problem

It is now needed to target a specific filename to get to a schema.:
```ts
import { UserUpdateInputObjectSchema } from '{...}/gen/schemas/objects/UserUpdateInput.schema'
```

## Suggested solution

It would be really awesome if you could generate indexes so that we can just import like this:
```ts
import { UserUpdateInputObjectSchema } from `{...}/gen/schemas`
```

**Comment by omar-dulaimi** [comment]
Also, don't forget to star the project :)

**Comment by omar-dulaimi** [comment]
Fixed with `0.1.4`
